README.md

```diff
@@ -1,4 +1,4 @@
-[![Stand With Ukraine](./banner2-direct.svg)](https://stand-with-ukraine.pp.ua/)
+[![Stand With Ukraine](./banner2-direct.svg)](https://stand-with-ukraine.pp.ua/) 
```

Comment:
Would have been nice to remove this unnecessary nonsense. 
Politics should be out of programming.